### ISSUE TITLE
Opensearch Dashboard 2.15.0 release

### ISSUE THREAD

**Issue by adrienyhuel** [fbaligand/kibana-enhanced-table#350]
Hello, 

Opensearch resolved an important issue about alerting in their recent 2.15.0 release.

I tried to install your plugin, by taking your 2.14 release and modifying the version in the JSON I side the zip, but it doesn't work.

I rebuilt the plugin using yarn, and it does work.
But surprisingly, the zip is more than twice bigger (4MB instead of 1.75MB)

Could you build a release for Opensearch 2.15.0 ?

**Comment by fbaligand** [comment]
Hi,
I just built and published a release for OSD 2.15.0:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-2.15.0.zip

Enjoy ;)

**Comment by bdovaz** [comment]
@fbaligand there is a new 2.16.0 release and I can't install this plugin without a new release.

Thanks!

**Comment by fbaligand** [comment]
Well, I don’t automate publication because I’m never sure that the new OSD version is compatible with enhanced table plugin’s current code.
That said, you can take the last package for 2.15.0, open opensearch_dashboards.json file and replace 2.15.0 by 2.16.0 and that’s all.

**Comment by bdovaz** [comment]
> Well, I don’t automate publication because I’m never sure that the new OSD version is compatible with enhanced table plugin’s current code. That said, you can take the last package for 2.15.0, open opensearch_dashboards.json file and replace 2.15.0 by 2.16.0 and that’s all.

I can't because in my Dockerfile I install it like this:

````Dockerfile
RUN /usr/share/opensearch-dashboards/bin/opensearch-dashboards-plugin install https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-2.15.0.zip
````

**Comment by fbaligand** [comment]
Hi,
I just tested the plugin version with OSD 2.16.0. 
Good news, it works well!
So I just published the package for OSD 2.16.0:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-2.16.0.zip

To answer your(s) suggestion(s):
- well, yes, it would be great to have automated tests and automatic package publishing. But it's not so easy. So at this point, I have no automated tests. I only have a script that automates package creation.
- then, a PR would not help. I mean: the code is exactly the same for OSD 1.0 to 2.10 (osd-1.0 branch), and the same for OSD 2.11 to latest. I just have a script that get the code from branch, and then build the package for targeted OSD version. The script essentially change "opensearchDashboardsVersion" in opensearch_dashboards.json, compile the code, and build the zip file.

**Comment by bdovaz** [comment]
Same problem with 2.17 release, 

Can you make a release for 2.17? Thank you.

**Comment by bdovaz** [comment]
@fbaligand new [2.17.1 release](https://hub.docker.com/layers/opensearchproject/opensearch-dashboards/2.17.1/images/sha256-928c597037b4c2ce9db3e1912976567cb9c09b5dd7910cf2164df582406ca963?context=explore), same problem.

I'm sorry to be a pain with this topic but as there is no way to help you (as far as I understand) I have no choice but to tell you, if it were in my power I would create a PR.

**Comment by bdovaz** [comment]
@fbaligand new 2.18 release.

Thanks!

**Comment by fbaligand** [comment]
Hi @bdovaz,

I just published enhanced table plugin package for OSD 2.18.0:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-2.18.0.zip

**Comment by bdovaz** [comment]
@fbaligand new 2.19 release.

Thanks!

**Comment by fbaligand** [comment]
Hi @bdovaz,

I just published the package for OSD 2.19.0:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-2.19.0.zip

Tell me if it works fine ;)

**Comment by bdovaz** [comment]
@fbaligand new 2.19.1 release.

Thanks!

**Comment by bdovaz** [comment]
@fbaligand new 2.19.2 release.

Thanks!

**Comment by fbaligand** [comment]
Hi @bdovaz,

I just published the package for 2.19.2:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-2.19.2.zip

**Comment by adrienyhuel** [comment]
Hello @fbaligand ,

OpenSearch just released 3.0.0 version.

Can you build a package for this major version ?

Thanks

**Comment by fbaligand** [comment]
No problem :)
I just published the package for OSD 3.0.0!
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-3.0.0.zip

By the way, happy to see that there is no breaking change in this OSD major version! (up to my tests)

**Comment by fbaligand** [comment]
Hi @adrienyhuel,

I just built the package for OpenSearch Dashboards 3.1.0!

It is available here:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-3.1.0.zip

**Comment by bdovaz** [comment]
@fbaligand new 2.19.3 release.

Thanks!

**Comment by fbaligand** [comment]
Hi @bdovaz,

I just built the package for OSD 2.19.3!

It is available here:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-2.19.3.zip

**Comment by fbaligand** [comment]
Hi, I just published enhanced table package for OSD 3.2.0:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-3.2.0.zip

**Comment by adrienyhuel** [comment]
Hello @fbaligand ,

OpenSearch released 3.3.0 version.
Can you build a package for this version ?

Thanks so much for all your work

**Comment by fbaligand** [comment]
Hi @adrienyhuel,

I just published the package for OSD 3.3.0:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-3.3.0.zip

**Comment by adrienyhuel** [comment]
Hello @fbaligand ,

OpenSearch released 3.3.1 version.
Can you build a package for this version ?

Thanks so much for all your work

**Comment by fbaligand** [comment]
Hi @adrienyhuel,

I just published the package for OSD 3.3.1:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-3.3.1.zip

**Comment by adrienyhuel** [comment]
Hello @fbaligand ,

OpenSearch released 3.3.2 version.
Can you build a package for this version ?

Thanks so much for all your work

**Comment by fbaligand** [comment]
Hi @adrienyhuel,

I just published the package for OSD 3.3.2:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-3.3.2.zip

**Comment by bdovaz** [comment]
@fbaligand 3.4.0 released.

Thanks!

**Comment by fbaligand** [comment]
Hi @bdovaz,

I just published the package for OSD 3.4.0:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_osd-3.4.0.zip

By the way, I also removed packages for OSD 3.3.1 and 3.3.2, given that OSD finally didn't publish 3.3.1 and 3.3.2 releases (neither on Docker Hub, nor on GitHub repository).
There are 3.3.1 and 3.3.2 releases for OpenSearch, but not for OpenSearch-Dashboards.

**Comment by bdovaz** [comment]
@fbaligand 3.5.0 released.

Thanks!

**Comment by fbaligand** [comment]
Hi @bdovaz, I just tested the plugin with OSD 3.5.0, and unfortunately, it doesn't work.
That implies that this OSD version requires changes in Enhanced Table plugin's code.

**Comment by bdovaz** [comment]
@fbaligand I have created the issue for traceability: https://github.com/fbaligand/kibana-enhanced-table/issues/373

**Comment by fbaligand** [comment]
Thanks

**Comment by fbaligand** [comment]
I just published the package for OSD 3.5.0!
You can download the package here:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.15.0/enhanced-table-1.15.0_osd-3.5.0.zip

**Comment by bdovaz** [comment]
@fbaligand 3.6.0 released.

Thanks!

**Comment by fbaligand** [comment]
I just published the package for OSD 3.6.0!
You can download the package here:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.15.0/enhanced-table-1.15.0_osd-3.6.0.zip

**Comment by bdovaz** [comment]
@fbaligand 3.7.0 released.

Thanks!

**Comment by fbaligand** [comment]
Hi @bdovaz,

I just published the package for OSD 3.7.0!
You can download the package here:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.15.0/enhanced-table-1.15.0_osd-3.7.0.zip

**Comment by bdovaz** [comment]
@fbaligand 3.8.0 released.

Thanks!

**Comment by fbaligand** [comment]
Hi @bdovaz,

I just published the package for OSD 3.8.0!
You can download the package here:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.15.0/enhanced-table-1.15.0_osd-3.8.0.zip
